lib/chef/chef_fs/file_system/repository/chef_repository_file_system_root_dir.rb

```diff
@@ -44,6 +44,7 @@
 require "chef/chef_fs/data_handler/user_data_handler"
 require "chef/chef_fs/data_handler/group_data_handler"
 require "chef/chef_fs/data_handler/container_data_handler"
+require "chef"
```

Comment:
this is most likely going to create dependency loops where chef requires chef_fs which requires chef.

i'd prefer to go back to simpler ruby code than trying to re-use chef provider code here, it's very likely to have unintended side effects.